app/helpers/offers_helper.rb

```diff
@@ -23,4 +23,44 @@ def category_list_classes depth, children, current_section
     children_class = (visible_children && depth <= 3) ? 'has-children' : ''
     "#{depth_class} #{children_class}"
   end
+
+  def offer_with_contacts offer
+    offer.contact_people.order('last_name ASC').order('first_name ASC')
+  end
+
+  def display_contacts? offer, contact
+    !offer.hide_contact_people || contact.spoc
+  end
+
+  def contact_name contact
+    if contact.last_name?
+      contact_gender(contact).to_s + contact_academic_title(contact).to_s + contact_full_name(contact).to_s
```

Comment:
Bitte auf Zeilenlängen achten. (max 80)
Tritt auch in anderen Dateien ein paar mal auf.